src/autofill-utils.js

```diff
@@ -589,24 +589,15 @@ function findElementsInShadowTree(root, selector) {
 
 /**
  * Default operation: finds elements using querySelectorAll.
- * Optionally, if forced an form.elements is iterable, attempts to find elements using .elements instead,
- * to catch field outside the form itsel using the form attribute. It also catches all elements
- * when the markup is broken. We use .filter along with a selector to avoid any unwanted elements.
  * Optionally, can be forced to scan the shadow tree.
  * @param {string} selector
  * @param {boolean} forceScanShadowTree
- * @param {boolean} shouldCheckFormControls
  * @returns {Element[]}
  */
-function getFormElements(form, selector, forceScanShadowTree = false, shouldCheckFormControls = false) {
+function queryFormElements(form, selector, forceScanShadowTree = false) {
```

Comment:
Please note this needs to be in the utils as it's also used outside of Form class.

Reply:
At this point this is function no longer has anything to do with forms, right? It's just a wrapper around `querySelectorAll` that also works with the shadow tree. Should we rename it?